app/src/main/java/com/tvtracker/MainActivity.kt

```diff
@@ -208,22 +209,29 @@ class MainActivity : ComponentActivity() {
                 Column() {
                     Row(modifier = Modifier
                         .fillMaxWidth()
-                        .background(Color(0xFFE0E0E0)), horizontalArrangement = Arrangement.SpaceBetween) {
+                        .background(Color(0xFFE0E0E0)),
+                        horizontalArrangement = Arrangement.SpaceBetween) {
                         //Close Button
-                        Button( onClick = { openDialog = false}, colors = ButtonDefaults.buttonColors(backgroundColor = Color.Transparent), elevation = ButtonDefaults.elevation(0.dp, 0.dp)) {
+                        Button( onClick = { openDialog = false},
+                            colors = ButtonDefaults.buttonColors(backgroundColor = Color.Transparent),
+                            elevation = ButtonDefaults.elevation(0.dp, 0.dp),
+                            
+                            ) {
                             Icon(
                                 Icons.Outlined.Close,
-                                contentDescription = "Close",
+                                contentDescription = stringResource(R.string.Close),
                                 modifier = Modifier.size(ButtonDefaults.IconSize + 16.dp),
                                 tint = Color.Black
                             )
                         }
 
                         //Favorites Button
-                        Button(onClick = { viewModel.saveMediaItem() }, colors = ButtonDefaults.buttonColors(backgroundColor = Color.Transparent), elevation = ButtonDefaults.elevation(0.dp, 0.dp)) {
+                        Button(onClick = { viewModel.saveMediaItem() },
+                            colors = ButtonDefaults.buttonColors(backgroundColor = Color.Transparent),
```

Comment:
Broke up UI properties for added readability to display different elements implementations, as well as to wrap lines that were too long.
https://developer.android.com/kotlin/style-guide#where_to_break